PackageExplorer/Interop/NativeDragDrop.cs

```diff
@@ -22,15 +22,18 @@ public static Stream CreateFileGroupDescriptorW(string fileName, DateTimeOffset
             var fileDescriptor = new FILEDESCRIPTORW() { cFileName = fileName };
             fileDescriptor.dwFlags |= FD_SHOWPROGRESSUI;
 
-            fileDescriptor.dwFlags |= FD_CREATETIME | FD_WRITESTIME;
-            var changeTime = lastWriteTime.ToLocalTime().ToFileTime();
-            var changeTimeFileTime = new System.Runtime.InteropServices.ComTypes.FILETIME
+            if (lastWriteTime != default(DateTimeOffset))
             {
-                dwLowDateTime = (int)(changeTime & 0xffffffff),
-                dwHighDateTime = (int)(changeTime >> 32),
-            };
-            fileDescriptor.ftLastWriteTime = changeTimeFileTime;
-            fileDescriptor.ftCreationTime = changeTimeFileTime;
+                fileDescriptor.dwFlags |= FD_CREATETIME | FD_WRITESTIME;
+                var changeTime = lastWriteTime.ToLocalTime().ToFileTime();
```

Comment:
Should this be local time or UTC for this API?

Reply:
That's some of the code I copied from http://dlaa.me/blog/post/9917797.

But looking at the source code of `DateTimeOffset.ToFileTime()` the call to `ToLocalTime()` is acutally unnecessary.

https://github.com/dotnet/corefx/blob/aeda6ab039f8d6e6041379ab156d51410b679c13/src/Common/src/CoreLib/System/DateTimeOffset.cs#L711-L714
or
https://referencesource.microsoft.com/#mscorlib/system/datetimeoffset.cs,621

I will remove it.